src/SFA.DAS.AssessorService.Web/Views/Banners/_GovNotifyBannerPartial.cshtml

```diff
@@ -1,7 +1,7 @@
 ﻿<div class="govuk-notification-banner" role="region"
      aria-labelledby="govuk-notification-banner-title"
      data-module="govuk-notification-banner">
-    <div class="govuk-notification-banner__header">
```

Comment:
With a content change like this need to confirm with PO and Content designer that the message is being removed